ftdetect/urscript.vim

```diff
@@ -18,4 +18,4 @@
 " You should have received a copy of the GNU General Public License
 " along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
-au BufRead,BufNewFile *.{ur,urs,urscript} set filetype=urscript
+au BufRead,BufNewFile *.{ur,urs,urscript,script} set filetype=urscript
```

Comment:
I'm a bit hesitant to add something as generic as "script" without any smarted detection. I think this is also why I didn't add it in the first place.

Is there a way we could make this smarter? :thinking: